### PR TITLE
chore: update tenants doc

### PR DESCRIPTION
**Pull request by gao-sun**
## Summary

provide an overview about tenants and related scenarios

### CODE REVIEW

File: docs/docs/references/tenants/README.md [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+        EC2 --- U4 -->|member of| O2
+        EC3 --- U5 -->|member of| O3
+    end
+```
+
+To automatically assign users to the correct organization, you can use [webhooks](/docs/recipes/webhooks/) to listen for user creation events and assign the user to the correct organization based on the enterprise connector.
+
+> We are planning to provide the "just-in-time provisioning" feature to automatically assign users to the correct organization based on the enterprise connector. Upvote this feature on our [roadmap](https://logto.productlane.com/roadmap) if you are interested.
+
+### I would like build a SaaS application with multi-tenancy
```
Comment (charIeszhao) [inline docs/docs/references/tenants/README.md:58]:
```suggestion
### I would like to build a SaaS application with multi-tenancy
```

File: docs/docs/references/tenants/README.md [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+
+The development tenant (dev tenant) is primarily intended for testing purposes and should not be utilized in a production environment. These tenants allow access to premium and paid features available in paid plans, free of charge and without requiring a subscription. However, there are certain [limitations](#limitations-of-development-tenants) that apply to development tenants.
+
+### Production
+
+The production tenant is where end-users access the live app and you might need a paid subscription. You can subscribe to the free, or pro plan to create a production tenant.
+
+:::note
+Please be aware that you can only specify your tenant types during creation; it is not possible to modify the tenant type once it has been created.
+:::
```
Comment (charIeszhao) [inline docs/docs/references/tenants/README.md:121]: Maybe we should put this before the development tenant paragraph (at L108). Putting it here makes me feel I could not upgrade my free plan to pro...

**Review by charIeszhao — APPROVED** [review]

File: docs/docs/references/tenants/README.md [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+
+import Availability from '@components/Availability';
+
+# 🗂️ Tenants
+
+<Availability cloud oss={false} />
+
+A tenant is an isolated environment where you can manage user identities, applications, and all other Logto resources.
+
+## Do I need more than one production tenant?
```
Comment (charIeszhao) [inline docs/docs/references/tenants/README.md:13]:
```suggestion
## Do I need more than one production tenants?
```

File: docusaurus.config.js [hunk 4]
```diff
@@ -245,14 +245,20 @@ const config = {
...
         // `existingPath` is the path `to`, the return value is the path `from`.
         createRedirects(existingPath) {
           if (existingPath.includes('/quick-starts')) {
             return existingPath.replace('/quick-starts', '/sdk');
           }
 
-          if (['/references/openid-connect', '/references/tenant-type'].some((path) => existingPath.includes(path))) {
-            return existingPath.replace('/references/', '/recipes/');
+          if (existingPath.includes('/references/openid-connect')) {
+            return existingPath.replace('/references/openid-connect', '/recipes/openid-connect');
```
Comment (charIeszhao) [inline docusaurus.config.js:261]: Can this be written as the simple "from" & "to" rule?

Comment (gao-sun) [reply docusaurus.config.js:261]: no, `openid-connect` has several pages

File: docs/docs/references/tenants/README.md (hunk 2 above)
Comment (gao-sun) [reply docs/docs/references/tenants/README.md]: cool updated

**Pull request merged**